src/test/java/eu/dissco/backend/repository/ElasticSearchRepositoryIT.java

```diff
@@ -187,7 +200,7 @@ void testGetLatestSpecimenSecondPage() throws IOException {
 
     // When
     var responseReceived = repository.getLatestSpecimen(pageNumber, pageSize);
-    var allElems = repository.getLatestAnnotations(1, pageSize * 2);
+    repository.getLatestAnnotations(1, pageSize * 2);
 
```

Comment:
Should delete this line altogether

Reply:
Wasn't sure and noticed nothing was happening with the result. But I guess removing a get won't fail the test. Will remove